crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -539,19 +577,12 @@ fn apply_pbr_lighting(
 
 #ifdef STANDARD_MATERIAL_DIFFUSE_TRANSMISSION
     transmitted_light += transmitted_environment_light.diffuse * diffuse_transmissive_color;
-#endif
+#endif  // STANDARD_MATERIAL_DIFFUSE_TRANSMISSION
 #ifdef STANDARD_MATERIAL_SPECULAR_TRANSMISSION
     specular_transmitted_environment_light = transmitted_environment_light.specular * specular_transmissive_color;
-#endif
-#endif // STANDARD_MATERIAL_DIFFUSE_OR_SPECULAR_TRANSMISSION
```

Comment:
I'm not sure exactly what's wrong here, but the changes to the ifdef breaks transmission. In the case of transmission the `let transmitted_environment_light` tries to sample the environment_map, but it's not behind the ifdef anymore so it blows up.

Reply:
I tried just wrapping that code with `#ifdef ENVIRONMENT_MAP` seems to work